Runtime/Behaviours/SendStartupMessage.cs

```diff
@@ -0,0 +1,73 @@
+using ReupVirtualTwin.helpers;
+using ReupVirtualTwin.behaviourInterfaces;
+using ReupVirtualTwin.dataModels;
+using ReupVirtualTwin.enums;
+using UnityEngine;
+using ReupVirtualTwin.helperInterfaces;
+
+
+namespace ReupVirtualTwin.behaviours
+{
+    [RequireComponent(typeof(IWebMessagesSender))]
+    public class SendStartupMessage: MonoBehaviour, ISendStartupMessage
+    {
+        public string buildVersion { get => _buildVersion; }
+        public IObjectMapper objectMapper { set => _objectMapper = value; }
+
+        string _buildVersion = "2024-04-05T19:56:03.550159";
```

Comment:
```suggestion
        string _buildVersion = "2024-04-05";
```
I'd go with just the date. We won't be making multiple breaking releases to prod on the same day